dla_cnn/desi/preprocess.py

```diff
@@ -225,10 +228,11 @@ def generate_summary_table(sightlines, output_dir, mode = "w"):
     Generate a csv file to store some necessary information of the given sightlines. The necessary information means the id, z_qso,
     s/n of thelymann forest part(avoid dlas+- 3000km/s), the wavelength range and corresponding pixel number of each channel. And the csv file's format is like:
     
-    id(int)， z_qso(float), s2n(float), wavelength_start_b(float), wavelength_end_b(float), pixel_start_b(int), pixel_end_b(int), wavelength_start_r(float), wavelength_end_r(float), pixel_start_r(int), pixel_end_r(int), wavelength_start_z(float), wavelength_end_z(float), pixel_start_z(int), pixel_end_z(int)
+    id(int)， z_qso(float), s2n(float), wavelength_start_b(float), wavelength_end_b(float), pixel_start_b(int), pixel_end_b(int), wavelength_start_r(float), wavelength_end_r(float), pixel_start_r(int), pixel_end_r(int), wavelength_start_z(float), wavelength_end_z(float), pixel_start_z(int), pixel_end_z(int),dlas_col_density(str),dlas_central_wavelength(str)
 
     "wavelength_start_b" means the start wavelength value of b channel, "wavelength_end_b" means the end wavelength value of b channel, "pixel_start_b" means the start pixel number of b channel, "pixel_end_b" means the end pixel number of b channel
-    so do the other two channels.
+    so do the other two channels.Besides, "dlas_col_density" means the col_density array of the sightline, and "dlas_central_wavelength" means the central wavelength array means the central wavelength array of the given sightline. Due to the uncertainty of the dlas' number, we chose to use str format to store the two arrays,
+    each array is written in the format like "value1,value2, value3", and one can use `str.split(",")` to get the data, the column density and central wavelength which have the same index in the two arrays corrspond to the same dla.
```

Comment:
I can understand why you did this (and probably had to), but
it will make using the Table for DLAs pretty difficult, if not impossible.

Maybe we just need a separate DLA table.
Anyhow, I'm ok with this for now and we can modify it later as need be.

Reply:
Sure.

Reply:
I will do it later.